wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -3053,16 +3053,31 @@ private function execute_alter() {
 				$this->rewriter->consume_all();
 			} elseif ( 'CHANGE' === $op_type ) {
 				// Parse the new column definition.
-				$raw_from_name    = 'COLUMN' === $op_subject ? $this->rewriter->skip()->token : $op_raw_subject;
-				$from_name        = $this->normalize_column_name( $raw_from_name );
-				$new_field        = $this->parse_mysql_create_table_field();
-				$alter_terminator = end( $this->rewriter->output_tokens );
-				$this->update_data_type_cache(
-					$this->table_name,
-					$new_field->name,
-					$new_field->mysql_data_type
+				$raw_from_name = 'COLUMN' === $op_subject ? $this->rewriter->skip()->token : $op_raw_subject;
```

Comment:
Is `ALTER TABLE <table-name> CHANGE COLUMN <column-name> SET DEFAULT <value>` valid MySQL syntax?

Based on the docs [here](https://dev.mysql.com/doc/refman/8.4/en/alter-table.html), it looks like the syntax should be:
```sql
ALTER TABLE table_name ALTER COLUMN column_name SET DEFAULT value;
```

And this seems to be correct in some testing:
```
MariaDB [149697504]> ALTER TABLE bptest CHANGE COLUMN stuff SET DEFAULT 777;
ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'SET DEFAULT 777' at line 1
MariaDB [149697504]> ALTER TABLE bptest ALTER COLUMN stuff SET DEFAULT 777;
Query OK, 0 rows affected (0.002 sec)
Records: 0  Duplicates: 0  Warnings: 0
```